commands/slash/attendanceof.go

```diff
@@ -0,0 +1,52 @@
+package slash
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/ritsec/ops-bot-iii/commands/slash/permission"
+	"github.com/ritsec/ops-bot-iii/logging"	
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+//Attendanceof slash command
+func Attendanceof() (*discordgo.ApplicationCommand, func(s *discordgo.Session, i *discordgo.InteractionCreate)) {
+	return &discordgo.ApplicationCommand {
+		Name: 		 			  "attendanceof",
+		Description: 			  "Get signin history of a user",
+		DefaultMemberPermissions: &permission.IGLead,
+		Options: []*discordgo.ApplicationCommandOption {
+			{
+				Type:        discordgo.ApplicationCommandOptionUser,
+				Name:        "user",
+				Description: "The user whose signin history you want to check",
+				Required:    true,
+			},
+		},
+	},
+	func(s *discordgo.Session, i *discordgo.InteractionCreate) {
+		span := tracer.StartSpan(
+			"commands.slash.attendanceof:Attendanceof",
+			tracer.ResourceName("/attendanceof"),
+		)
+		defer span.Finish()
+
+		u := i.ApplicationCommandData().Options[0].UserValue(s)
+
+		logging.Debug(s, "Attendanceof command received for " + u.Username, i.Member.User, span)
+
+		err := s.InteractionRespond(
+			i.Interaction,
+			&discordgo.InteractionResponse {
+				Type: discordgo.InteractionResponseChannelMessageWithSource,
+				Data: &discordgo.InteractionResponseData {
+					Content: attendanceMessage(u.ID, span.Context()),
```

Comment:
the attendanceMessage function is returning the "Your Signins:" when it is not my signins if I am querying on someone.
![image](https://github.com/user-attachments/assets/358831a6-1c1c-478f-8ed0-f9128119b9aa)